python/ray/util/client/server/server.py

```diff
@@ -377,20 +379,37 @@ def _async_get_object(
             with disable_client_hook():
 
                 def send_get_response(result: Any) -> None:
-                    """Pushes a GetResponse to the main DataPath loop to send
+                    """Pushes GetResponses to the main DataPath loop to send
```

Comment:
How much change would it be to consolidate sync and async `get()`? Maintaining two implementations of get on client and server with chunking and reconnection seems like a burden later on.

Reply:
I suspect it shouldn't be too bad, opened #22357 to track this